Dockerfile

```diff
@@ -14,7 +14,10 @@ ENV JAVA_OPTS="${JAVA_OPTS} -Dspring.profiles.active=default"
 WORKDIR /app
 ARG DEPENDENCY=/app
 COPY --from=builder $DEPENDENCY/dependencies/ ./
+RUN true
```

Comment:
why?

Reply:
Weird issue, but known to exist since 2018-ish. Using a `RUN true` between `COPY`s was a suggested solution to the build failures, and worked for us.

https://github.com/moby/moby/issues/37965

Reply:
🙄🙄🙄🙄🙄
ok, document with a link to that issue on the first RUN true line.

Reply:
added comments per your request